modules/@apostrophecms/util/ui/src/http.js

```diff
@@ -147,7 +148,7 @@ export default () => {
     let i;
 
     if (options.qs) {
-      url = apos.http.addQueryToUrl(url, options.qs);
+      url = url ? apos.http.addQueryToUrl(url, options.qs) : '/';
```

Comment:
we should set the `url` at the beginning of the function to avoid weird edge cases